### PR TITLE
[grafana] Add networkpolicy support

### PR DESCRIPTION
**Pull request by hicwic**
### Description of the change

Add networkpolicy support on grafana chart.
Networkpolicy control access to HTTP port.

If the chart is deployed on a K8S cluster where default rule is deny all, we need to deployed a networkpolicy but it can be tedious. we want to have a chart align with other ones to integrate a default networkpolicy.

### Possible drawbacks

By default network policy is disable, but when it's enabled some change must be done to be work fine.
the good label and sometime the right namespaces selector must be added to each pod, want to reach grafana.

### Applicable issues

fixes #

### Checklist

- [x] Chart version bumped in Chart.yaml according to [semver](https://semver.org/).
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by zanhsieh** [comment]
@hicwic 
Could you sign DCO please?

**Comment by hicwic** [comment]
Never did that before... learning every day.  Hope this is good for you now.

**Review by zanhsieh — APPROVED** [review]

File: charts/grafana/values.yaml [hunk 1]
```diff
@@ -760,3 +760,29 @@ imageRenderer:
...
+  ## LabelSelector to select these namespaces, note that the networkPolicy's namespace should also be explicitly added.
+  ##
+  ## Example:
+  ## explicitNamespacesSelector:
+  ##   matchLabels:
+  ##     role: frontend
+  ##   matchExpressions:
+  ##    - {key: role, operator: In, values: [frontend]}
+  ##
+  explicitNamespacesSelector: {}
```
Comment (zanhsieh) [inline charts/grafana/values.yaml:788]: Please add a new line character at the end of file.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/grafana/values.yaml (hunk 1 above)
Comment (hicwic) [reply charts/grafana/values.yaml]: done

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**